server/src/main/java/com/linecorp/centraldogma/server/internal/storage/project/DefaultProject.java

```diff
@@ -185,9 +186,20 @@ private RepositoryManager newRepoManager(File rootDir, Executor repositoryWorker
     private void createReservedRepos(long creationTimeMillis, boolean useDogmaRepoAsMetaRepo) {
         if (!repos.exists(REPO_DOGMA)) {
             try {
-                repos.create(REPO_DOGMA, creationTimeMillis, Author.SYSTEM);
+                final Repository dogmaRepository =
+                        repos.create(REPO_DOGMA, creationTimeMillis, Author.SYSTEM);
+                if (useDogmaRepoAsMetaRepo) {
+                    dogmaRepository.commit(
+                            Revision.HEAD, creationTimeMillis, Author.SYSTEM,
+                            "Add " + META_TO_DOGMA_MIGRATED + " file to dogma repository", "", Markup.PLAINTEXT,
+                            Change.ofJsonUpsert(META_TO_DOGMA_MIGRATED,
+                                                Jackson.writeValueAsString(
+                                                        ImmutableMap.of("timestamp", Instant.now())))).join();
```

Comment:
`CreateProjectCommand` is executed in each cluster. Does that mean `Change.ofJsonUpsert(META_TO_DOGMA_MIGRATED)` could raise `RedundantChangeException`?

Reply:
> Change.ofJsonUpsert(META_TO_DOGMA_MIGRATED) could raise RedundantChangeException?

I don't think so. This happens only after the migration job is completed. After the migration job is completed,
each replica creates its own dogma repository with this commit.

Reply:
Understood now. I was confused. The commit was submitted to a command executor.